.circleci/scripts/binary_linux_build.sh

```diff
@@ -7,7 +7,7 @@ source /env
 # Defaults here so they can be changed in one place
 # This script is run inside Docker.2XLarge+ container that has 20 CPU cores
 # But ncpu will return total number of cores on the system
-export MAX_JOBS=18
+export MAX_JOBS=$(( $(nproc) - 2 ))
```

Comment:
I thought `nproc` was suppose to return # of available CPU on the host instead of the container? @malfet

Reply:
We had a conversation about it and we think the issue with the jobs stemmed from the use of `stdbuf` and not necessarily from the `nproc` invocation

Reply:
@seemethere can you please delete comments on lines 8-9 then?
@walterddr yes, that is correct, but actually spawning more build jobs than number of cores is fine.